src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1863,21 +1921,6 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
     });
 
-    /*readonly mutationProfileDuplicateSamplesReport = remoteData({
```

Comment:
lets leave this in and add comment that "in case we want to handle duplicate samples differently"